msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -41,28 +42,28 @@
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp1.0' ">
     <DefineConstants>$(DefineConstants);NET_CORE</DefineConstants>
-  </PropertyGroup>
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_0;DOTNET;PORTABLE;HAVE_ASYNC;HAVE_COVARIANT_GENERICS;HAVE_DATA_CONTRACTS;HAVE_DATE_TIME_OFFSET;HAVE_FSHARP_TYPES;HAVE_GUID_TRY_PARSE;HAVE_HASH_SET;HAVE_IGNORE_DATA_MEMBER_ATTRIBUTE;HAVE_INOTIFY_COLLECTION_CHANGED;HAVE_ISET;HAVE_LINQ;HAVE_METHOD_IMPL_ATTRIBUTE;HAVE_READ_ONLY_COLLECTIONS;HAVE_REFLECTION_BINDER;HAVE_SERIALIZATION_BINDER_BIND_TO_NAME;HAVE_STRING_JOIN_WITH_ENUMERABLE;HAVE_TIME_SPAN_PARSE_WITH_CULTURE;HAVE_TIME_SPAN_TO_STRING_WITH_CULTURE;HAVE_TIME_ZONE_INFO;HAVE_TYPE_DESCRIPTOR;HAVE_UNICODE_SURROGATE_DETECTION;HAVE_VARIANT_TYPE_PARAMETERS;HAVE_VERSION_TRY_PARSE;HAVE_OBSOLETE_FORMATTER_ASSEMBLY_STYLE;$(AdditionalConstants)</DefineConstants>  </PropertyGroup>
```

Comment:
Hmm, Where were these picked up from ? The whole "flags for features" are ok, but I am hesitant about adding the flag NETSTANDARD1_3 to Xamarin.iOS (and the other "platform" flags, but this one in particular is worrisome). This is because currently we associate a platform to a target framework against which we build our assembly. Adding new flags breaks this association (mental association and in the case of iOS == netstandard1_3, also real association) - how will we be able to define an object that lives in the netstandard1.3 version of our DLL but not in iOS (and vice versa) ?

Reply:
OK.  I've fixed these to use our defines as necessary and removed the netstandard checks.

Reply:
LGTM!